src/components/staking-validators/staking-validators.tsx

```diff
@@ -55,9 +63,22 @@ export const StakingValidators = ({
       <SectionList
         style={styles.container}
         renderItem={({item}) => <ValidatorRow item={item} onPress={onPress} />}
-        renderSectionHeader={({section: {title}}) => (
-          <SectionHeader title={title} />
-        )}
+        renderSectionHeader={({section: {title}}) => {
+          return (
+            <View>
+              {/*When validators array has only one object it means only one unstacked section.
+              In this case title should not be shown*/}
+              {validators.length > 1 && <SectionHeader title={title} />}
+              {/*Sort by random hint section should be shown only for unstacked validators*/}
+              {title === getText(I18N.stakingValidatorsUnStaked) && (
+                <View style={styles.sortSectionContainer}>
+                  <Icon name={IconsName.arrow_sort} color={Color.textBase1} />
+                  <Text t9 i18n={I18N.byRandomTitle} style={styles.sortTitle} />
```

Comment:
Fix this header
![image](https://github.com/haqq-network/haqq-wallet/assets/25716589/b2a81223-b85a-4582-a194-f0ec4552d2df)

Reply:
Overlap fixed